e2e/runJest.ts

```diff
@@ -18,6 +18,7 @@ import {normalizeIcons} from './Utils';
 const JEST_PATH = path.resolve(__dirname, '../packages/jest-cli/bin/jest.js');
 
 type RunJestOptions = {
+  preserveSymlinks?: string;
```

Comment:
using `nodeOptions` should be sufficient here without adding a new option

Reply:
nodeOptions actually doesn't work here